src/bundle/Resources/public/scss/_balloon-form.scss

```diff
@@ -91,7 +91,12 @@
                 }
             }
 
+            &.ck-button-revert {
```

Comment:
It could be done on `+` selector to catch all future additions on new buttons, but that's a nitpick for me for now